src/store/wizardSlice.ts

```diff
@@ -376,6 +381,9 @@ export const wizardSlice = createSlice({
     ) => {
       state.fileSystem.partitions = action.payload;
     },
+    changeIsNextButtonPristine: (state) => {
+      state.fileSystem.isNextButtonPristine = false;
+    },
```

Comment:
Could there be a use case for needing to set `isNextButtonPristine` to true again? I'd say let's either set the value based on action.payload or we can rename the function so the name fits the functionality a bit better. Something like `setIsNextButtonPristineToFalse`, what do you think?